build.gradle

```diff
@@ -86,5 +87,5 @@ subprojects {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '4.6'
+    gradleVersion = '4.7'
```

Comment:
When you updated this, did you also run: `./gradlew wrapper` from the command line?

Reply:
Let me include the files.